src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +13,31 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    static final char LETTER = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String text = null;
+        try {
+            text = Files.readString(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        if (text.length() == 0) {
```

Comment:
try to create a solution without this check

Reply:
Fixed.